idaes/surrogate/pysmo/sampling.py

```diff
@@ -720,29 +817,23 @@ def __init__(self, data_input, number_of_samples=None, sampling_type=None):
         print('Sampling type: ', self.sampling_type, '\n')
```

Comment:
print -> log

Reply:
For now PySMO uses print statements throughout the entire package. I'd like to keep it consistent for now.

Changing all the print statements in PySMO to logs will a separate undertaking and will be addressed in a separate PR later.